pkg/networkserver/config.go

```diff
@@ -26,7 +26,8 @@ import (
 type Config struct {
 	Devices             DeviceRegistry         `name:"-"`
 	DownlinkTasks       DownlinkTaskQueue      `name:"-"`
-	NetID               types.NetID            `name:"net_id" description:"NetID of this Network Server"`
+	NetID               types.NetID            `name:"net-id" description:"NetID of this Network Server"`
+	DevAddrPrefixes     []types.DevAddrPrefix  `name:"dev-addr-prefixes" description:"DevAddrPrefixes of this Network Server"`
```

Comment:
Let's spell this out; `Device address prefixes`, in contrary to `NetID` which is a LoRaWAN term